build.gradle

```diff
@@ -1,243 +1,232 @@
-                }
-                usage('java-runtime') {
-                    fromResolutionResult()
-                }
-            }
+        testImplementation 'org.assertj:assertj-core:3.24.2'
+        testImplementation enforcedPlatform('org.junit:junit-bom:5.9.2')
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+        testImplementation 'org.junit.jupiter:junit-jupiter-api'
 
-            pom {
-                name = project.name
-                description = 'pg-index-health is a Java library for analyzing and maintaining indexes health in PostgreSQL databases.'
-                url = 'https://github.com/mfvanek/pg-index-health'
-                licenses {
-                    license {
-                        name = 'Apache License Version 2.0'
-                        url = 'https://www.apache.org/licenses/LICENSE-2.0'
-                    }
-                }
+        checkstyle 'com.thomasjensen.checkstyle.addons:checkstyle-addons:7.0.1'
+    }
 
-                developers {
-                    developer {
-                        id = 'mfvanek'
-                        name = 'Ivan Vakhrushev'
-                        email = 'mfvanek@gmail.com'
-                    }
-                }
+    java {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+        withJavadocJar()
+        withSourcesJar()
+    }
 
-                scm {
-                    connection = 'scm:git:https://github.com/mfvanek/pg-index-health.git'
-                    developerConnection = 'scm:git@github.com:mfvanek/pg-index-health.git'
-                    url = 'https://github.com/mfvanek/pg-index-health'
-                }
-            }
+    jar {
+        manifest {
+            attributes("Implementation-Title": project.name,
+                    "Implementation-Version": project.version)
         }
     }
 
-    repositories {
-        maven {
-            def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2"
-            def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots"
-            url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
-            credentials {
-                username sonatypeUsername
-                password sonatypePassword
-            }
+    javadoc {
+        if (JavaVersion.current().isJava9Compatible()) {
+            options.addBooleanOption('html5', true)
         }
     }
-}
 
-signing {
-    if (!version.endsWith('SNAPSHOT')) {
-        useGpgCmd()
-        sign publishing.publications.mavenJava
+    checkstyle {
+        toolVersion '10.5.0'
+        configFile file("../config/checkstyle/checkstyle.xml")
+        ignoreFailures = false
+        maxWarnings = 0
+        maxErrors = 0
     }
-}
-
-javadoc {
-    if (JavaVersion.current().isJava9Compatible()) {
-        options.addBooleanOption('html5', true)
+//    checkstyleMain {

```

Comment:
Better to remove if it's not needed. If this is necessary for some reason, then leave an explanatory comment.

Reply:
Fixed